pyaedt/modules/SolveSetup.py

```diff
@@ -1787,12 +1787,14 @@ def create_frequency_sweep(
 
         Parameters
         ----------
-        unit : str
-            Frequency Units.
-        freqstart : float
+        unit : str, optional
+            Frequency Units. Default is ``"GHz"``.
+        freqstart : float, str, optional
             Starting frequency of the sweep, such as ``1``.
-        freqstop : float
-            Stopping frequency of the sweep.
+            If units is passed with number like ``"1MHz"`` then the unit will be ignored.
+        freqstop : float, str, optional
+            Stopping frequency of the sweep. Default is ``10``.
+            If units is passed with number like ``"1MHz"`` then the unit will be ignored.
```

Comment:
```suggestion
            If a unit is passed with number, such as ``"1MHz"``, the unit will be ignored.
        freqstop : float, str, optional
            Stopping frequency of the sweep. The default is ``10``.
            If a unit is passed with number, such as ``"1MHz"`, the unit is ignored.
```
Use present rather than future tense.